src/tests/ftest/control/dmg_system_start.yaml

```diff
@@ -0,0 +1,25 @@
+hosts:
+  test_servers: 1
+server_config:
+  name: daos_server
+  engines_per_host: 2
```

Comment:
This new test is working so not an issue. But just FYI this is now the only test using dual-engine on VMs (sharing `fabric_iface: eth0`). The only issue I could think of is that on our VMs, `targets: 4` * 2 might overload the VMs. There is only one other VM test using a total of 8 targets and it sets `DAOS_TARGET_OVERSUBSCRIBE=1`.  But this test didn't seem to need that.
Anyway, not an issue unless we run into something in the future. IMO it's great to user fewer nodes when possible

Reply:
Hmm, okay. I actually wrote the file this way entirely so it would run on my solitary node in the lab. It doesn't necessarily need to be two engines on a single node, that was just all I had access to.

Reply:
Good point. IMO it's great to do that because it makes it easier for people to test locally